system-addon/common/Reducers.jsm

```diff
@@ -31,6 +31,7 @@ const INITIAL_STATE = {
     initialized: false,
     values: {}
   },
+  Theme: {className: ""},
```

Comment:
I *think* the way it will probably work is there will be a list of css variables here that we expose to themes + web extensions so they can override. So yeah, I envision more properties added here.

Reply:
And I think we'll probably keep the feed around but it might be different? I haven't thought much about how we would make newtab themeable on a window by window basis. It depends on the implementation details of bug 1444459 I guess?